### PR TITLE
Sis imports extra info

### PR DESCRIPTION
**Pull request by exacs**
- The package is "upgraded" to version `1.2.0-0` (release candidate version)
- Upgrade to `1.2.0` when merging to master (if it's ok)

### CODE REVIEW

File: sis_import_utils/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kth/sis_import_utils",
-  "version": "1.1.1",
+  "version": "1.2.1-0",
```
Comment (Delodax) [inline sis_import_utils/package.json]: What is the purpose of the extra hyphen and number in the version?

**Comment by Delodax** [comment]
What is the purpose of the hyphen and number in the version?

File: sis_import_utils/index.js [hunk 2]
```diff
@@ -37,65 +38,137 @@ async function getUserInfo (userId, ldapClient) {
...
+    const ugPassword = data.ugPassword || process.env.UG_PASSWORD
+
+    const ldapClient = ldap.createClient({ url: ugUrl })
+    const ldapBind = util.promisify(ldapClient.bind).bind(ldapClient)
+    const ldapUnbind = util.promisify(ldapClient.unbind).bind(ldapClient)
+    let userInfo
+    try {
+      await ldapBind(ugUsername, ugPassword)
+      userInfo = await getUserInfo(userId, ldapClient)
+
```
Comment (Delodax) [inline sis_import_utils/index.js]: Unneeded empty line.

**Comment by exacs** [comment]
Before merging, verify that:

- lms-scripts (collect_…) is working with this new version without changing anything
- build-monitor is working with this new version without changing anything

If that works, upgrade the package version to `1.2.0`.

**Comment by exacs** [comment]
Before merging, verify that:

- lms-scripts (collect_…) is working with this new version without changing anything
- build-monitor is working with this new version without changing anything

If that works, upgrade the package version to `1.2.0`.

File: sis_import_utils/index.js [hunk 3]
```diff
@@ -37,65 +38,137 @@ async function getUserInfo (userId, ldapClient) {
...
+      )
+    }
+  }, options)
+
+  return result.join('\n')
 }
 
 module.exports = {
-  getFilteredErrors: getFilteredErrors
+  getFilteredErrors: getFilteredErrors,
```
Comment (exacs) [inline sis_import_utils/index.js]:
```suggestion
  getFilteredErrors,
```

Comment (exacs) [inline sis_import_utils/index.js]:
```suggestion
  getFilteredErrors,
```

**Pull request merged**